README.md

```diff
@@ -64,7 +64,7 @@ Resources
 -------------------------------
 
 * Website
-  * [www.nomadproject.io](https://www.nomadproject.io)
+  * [www.nomadproject.io](https://www.nomadproject.io) 
```

Comment:
😲

Reply:
Hah, that was just to re-trigger the CircleCI pipeline. I'll reverse it.